aws/logs_monitoring/lambda_function.py

```diff
@@ -390,14 +399,24 @@ def datadog_forwarder(event, context):
 
     if DD_FORWARD_LOG:
         forward_logs(filter_logs(map(json.dumps, logs)))
+        DD_FORWARDER_TELEMETRY["logs_forwarded"]+=len(list(filter_logs(map(json.dumps, logs))))
```

Comment:
I don't think it's a good idea to duplicate `filter_logs(map(...` here. Can you move logic into `forward_logs`, where you have filtered logs (and actual count)?

Reply:
Hi Tian - moved to `forward_logs` and removed duplication of the `filter_logs` call